api/tests/app/email_branding/test_rest.py

```diff
@@ -13,11 +13,16 @@ def test_get_email_branding_options(admin_request, notify_db, notify_db_session)
         'email_branding.get_email_branding_options'
     )['email_branding']
 
-    assert len(email_branding) == 2
+    assert 6 == len(email_branding)
```

Comment:
Any reason to change assert order? I suspect most of the codebase is using `actual, expected`, so I'd just leave these as is.

Reply:
Hmmm...I was getting an output like `Expected:6 Actual:2` with the expected value on the left hand side of the `==`, however seems like this is only for my IDE. I can change the order around again.